src/server/pkg/bloom/bloom_test.go

```diff
@@ -0,0 +1,167 @@
+package bloom
+
+import (
+	"crypto/rand"
+	mathrand "math/rand"
+	"testing"
+
+	"github.com/pachyderm/pachyderm/src/client/pkg/require"
+	"github.com/pachyderm/pachyderm/src/server/pkg/storage/hash"
+)
+
+func maxValue(f *BloomFilter) int {
+	max := 0
+	for _, value := range f.Buckets {
+		if int(value) > max {
+			max = int(value)
+		}
+	}
+	return max
+}
+
+// Uses the storage layer hash
+func makeHash(t *testing.T) []byte {
+	data := make([]byte, 128)
+	n, err := rand.Read(data)
+	require.NoError(t, err)
+	require.Equal(t, 128, n)
+	return hash.Sum(data)
+}
+
+func requirePanic(t *testing.T, cb func()) {
+	defer func() {
+		r := recover()
+		require.NotNil(t, r)
+	}()
+
+	cb()
+}
```

Comment:
Might as well just move this into the `require` pkg, we'll probably need it elsewhere eventually.